src/GraphQL.Tests/Bugs/Bug1136.cs

```diff
@@ -0,0 +1,21 @@
+using GraphQL.Language.AST;
+using GraphQL.Types;
+using Shouldly;
+using System.Linq;
+using Xunit;
+
+namespace GraphQL.Tests.Bugs
+{
+    public class Bug1136
+    {
+        [Fact]
+        public void IsValidLiteralValue_Should_Not_Throw_NRE()
+        {
+            var type = new DateGraphType() { Name = null };
```

Comment:
@joemcbride thoughts on this? should a graphtype ever be allowed to have a null name?

Reply:
No it shouldn’t.  Perhaps a better solution would be to throw an error when the schema is built if the name is null.

Reply:
> Perhaps a better solution would be to throw an error when the schema is built if the name is null.

Of course it will be! But this is a completely different story and not only the name should be checked. For example see #1156. Now there are many opportunities for carelessness to create such a schema that will be invalid. I'm sure this is the subject of a **separate task**. In general, now API of schema classes (GraphTypes) allows you to set any properties you like, and the problems will be revealed already at the stage of request execution.

Also note that the test does not use a schema at all. That is, protection when building a schema is required, but this is not a generally sufficient condition.